src/main/java/uk/gov/hmcts/cmc/claimstore/courtfinder/CourtFinderApi.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.cmc.claimstore.courtfinder;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import uk.gov.hmcts.cmc.claimstore.courtfinder.models.Court;
+import uk.gov.hmcts.cmc.claimstore.courtfinder.models.CourtDetails;
+
+import java.util.List;
+
+@FeignClient(name = "court-finder-api", url = "https://courttribunalfinder.service.gov.uk")
+public interface CourtFinderApi {
```

Comment:
consider adding a wiremock test proving this client works,
something like:
https://github.com/hmcts/idam-java-client/blob/master/src/test/java/uk/gov/hmcts/reform/idam/client/IdamClientTest.java
https://github.com/hmcts/idam-java-client/tree/master/src/test/resources/mappings